tests/conftest.py

```diff
@@ -427,3 +427,29 @@ def tag_test_report(request, pytestconfig, testbed, duthost, record_testsuite_pr
         record_testsuite_property("os_version", duthost.os_version)
 
         __report_metadata_added = True
+
+@pytest.fixture(scope="module", autouse=True)
+def disable_container_autorestart(duthost, request):
+    skip = False
+    for m in request.node.iter_markers():
+        if m.name == "enable_container_autorestart":
```

Comment:
I don't think you need to skip for this test. This test will enable auto-restart during execution.

Reply:
Maybe I don't fully understand your comment, but I believe the intention is this marker will be used to disable the disabling of container auto-restart. Since this fixture is marked as `autouse=True`, there needs to be some way to tell it "I don't want to disable container auto-restart".

Reply:
@jleveque I believe your understanding is correct.